NGitLab/IIssueClient.cs

```diff
@@ -55,6 +55,16 @@ public interface IIssueClient
 
         GitLabCollectionResponse<Issue> GetAsync(int projectId, IssueQuery query);
 
+        /// <summary>
+        ///     <para>Return a single issue, only for administrators.</para>
+        ///     <para>url like GET /issues/:id</para>
+        /// </summary>
+        /// <param name="issueId"></param>
+        /// <returns>The issue that corresponds to the issue id</returns>
+        Issue GetById(int issueId);
```

Comment:
Bonus: can you update the method `Issue Get(int projectId, int issueId);` => It's an issueIid (`Issue Get(int projectId, int issueIid);`

Reply:
Thanks for the feedback, regarding your first comment I received the following warning when I originally tried adding an overload for `GetAsync`:

> Warning RS0026 Symbol 'GetAsync' violates the backcompat requirement: 'Do not add multiple overloads with optional parameters'. See 'https://github.com/dotnet/roslyn/blob/main/docs/Adding%20Optional%20Parameters%20in%20Public%20API.md' for details.

I was unsure how to tackle this and was reluctant to introduce an issue so went with an entirely different method name, if there's something I can do to resolve this let me know and I'll update it.

I'll get to adding those tests 😃 !

Reply:
Oh ok, it's probably due to `GetAsync(int projectId, int issueId, CancellationToken cancellationToken = default);`

I suppose that if you don't put the default value to the cancellation token it will pass. `Task<Issue> GetByIdAsync(int issueId, CancellationToken cancellationToken);`

But on the rest of the method we always pass `= default`

Do you have a suggestion @meziantou for that kind of case?

Reply:
Ok :)

Reply:
@Ditchfuuu , you can concentrate on the Bonus then :)